flusher_test.go

```diff
@@ -173,14 +174,16 @@ func testFlushTraceLightstep(t *testing.T, protobuf, jsn io.Reader) {
 	server := setupVeneurServer(t, config, nil)
 	defer server.Shutdown()
 
-	lightstepTracer := lightstep.NewTracer(lightstep.Options{
-		AccessToken: "TestAccessToken",
-	})
+	lightstepTracerThunk := func() opentracing.Tracer {
+		return lightstep.NewTracer(lightstep.Options{
```

Comment:
I spent an embarrassing amount of time staring at this trying to figure out how in the hell this worked, only later realizing it was in the trace _test_ and therefore not the "thunker" that we use in real life. That one is `configureLightstepTracer`. :)